src/test/scala/strawman/collection/test/Test.scala

```diff
@@ -288,6 +288,12 @@ class StrawmanTest {
     println(xs17.to(List))
   }
 
+  def sortedSets(xs: strawman.collection.immutable.SortedSet[Int]): Unit = {
+    val xs1 = xs.map((x: Int) => x.toString)
+    val xs2: SortedSet[String] = xs1
+    val xs3: strawman.collection.immutable.Set[String] = xs.map((x: Int) => x.toString)
```

Comment:
This looks wrong. Scala 2.12 should be able to perform overload resolution correctly without a type annotation.